QUICKSTART.md

```diff
@@ -0,0 +1,293 @@
+# **KCP-Edge** Quickstart
+
+## Required Packages:
+   - [docker](https://docs.docker.com/get-docker/)
+   - [kind](https://kind.sigs.k8s.io/)
+   - kubectl (version range expected: 1.23-1.25)
+
+## Setup Instructions
+
+Table of contents:
+
+- [1. Install and run **KCP-Edge**](#1-install-kcp-edge-pre-requisites)
+- [2. Example deployment of nginx workload into a kind local cluster](#4-Example-deployment-of-nginx-workload-into-a-kind-local-cluster)
+  - [a. Stand up a local florin kind cluster](#a-Stand-up-a-local-florin-kind-cluster)
+  - [b. Create a sync target, placement, and edge syncer for onboarding the created florin edge cluster](#b-Create-a-sync-target-placement-and-edge-syncer-for-onboarding-the-created-florin-edge-cluster)
+  - [c. Create the nginx workload and deploy it to the florin cluster](#c-Create-the-nginx-workload-and-deploy-it-to-the-florin-cluster)
+- [3. Cleanup the environment](#5-Cleanup-the-environment)
+
+
+This guide is intended to show how to quickly bring up a **KCP-Edge** environment with its dependencies from a binary release.
+
+## 1. Install and run **KCP-Edge**
```

Comment:
My suggestion is not to include KCP in the title of the section to avoid highlighting it, given that the focus of the doc in on the KubeStellar. I added the suggested explanation below that we need kcp in order to run KubeStellar.

Reply:
Completely agree